packages/btc/src/address.ts

```diff
@@ -28,7 +28,7 @@ export function isSupportedFromAddress(address: string) {
  */
 export function publicKeyToPayment(publicKey: string, addressType: AddressType, networkType: NetworkType) {
   if (!publicKey) {
-    return void 0;
+    return undefined;
```

Comment:
It should be better to throw an exception than return `undefined`, so that the caller can realize the problem early.

Reply:
Yes, this part of the code is not aligned with the the other functions in the `address.ts`. Some return `undefined` while some don't, I'll check them out and modify in another PR, if nessesary.